frontend/src/UnitCounter.tsx

```diff
@@ -4,6 +4,7 @@ import Typography from "@mui/joy/Typography";
 import Stack from "@mui/joy/Stack";
 import { Unit } from "./data";
 import { InputStepper } from "./InputStepper";
+import TradeGood from "./assets/trade-good.png";
```

Comment:
Nit: `tradeGoodPath` maybe? That was the convention I used in `data.ts` but I wasn't exactly sure what to call it, so not married to it. But since it's the image path and not the image itself, then thinking camel case instead of pascal case?